src/setup.js

```diff
@@ -0,0 +1,5 @@
+const { User } = require('./models/User.model');
+const { Expense } = require('./models/Expense.model');
+
+User.sync({ force: true });
+Expense.sync({ force: true });
```

Comment:
Using `sync({ force: true })` will drop the table if it already exists and recreate it. This will result in data loss. Ensure this is the intended behavior, especially in a production environment. Consider using `sync()` without the `force` option if you want to preserve existing data.